package.json

```diff
@@ -37,14 +37,15 @@
     "jscs": "^3.0.7",
     "mocha": "^3.1.2",
     "nsp": "^2.6.2",
-    "restify": "^7.0.0",
     "restify-clients": "^1.4.0",
     "uuid": "^2.0.3"
   },
   "dependencies": {
     "ajv": "^4.8.0",
     "assert-plus": "^1.0.0",
+    "bunyan": "^1.8.12",
     "lodash": "^4.16.4",
+    "restify": "^7.2.0",
```

Comment:
Once we use child loggers I don't think we'll need bunyan anymore. What's restify needed for? Or is that really a devDependency?

Reply:
Restify exports the new `helpers.compose` function to create a handler chain, which is used in the `reloadProxy function`.